slack-gipfeli-bot/commands/add.rb

```diff
@@ -13,18 +13,27 @@ class Calculate < SlackRubyBot::Commands::Base
 
       command 'add' do |client, data, _match|
         # list = File.open('gipfeli_list', 'a')
+        cur_list, times, orders, new_order = cache.get('list'), [], [], match.to_s[12..-1]
 
-        wish = _match.to_s[12..-1]
-        if wish.nil?
-          client.say(channel: data.channel, text: 'You need to tell me your order! (e.g. \'add gipfeli\'')
+        cur_list.split("\n").each do |item|
+          times << e.match(/\d+x/).to_s
+          orders << e.match(/ [\S\W]+/).to_s[1..-1]
+        end
+
+        index = orders.map(&:downcase).find_index(new.downcase)
+
+        if new_order.nil?
+          client.say(channel: data.channel, text: 'You need to tell me your order! (e.g. \'add gipfeli\')')
+        elsif index
+          times[index] = "#{times[index][0..-2].to_i + 1}x"
```

Comment:
It may be useful to output the new amount…

Reply:
I don't think it matters to the end user if somebody else already ordered the same thing. If there is a demand for this it can be added in a later feature.